src/Web/appsettings.json

```diff
@@ -17,14 +17,13 @@
             }
         }
     },
-    "Scheduler": {
-	    "Driver": "nomad"
-    },
-    "Nomad": {
-        "Datacenters": [
-            "dc1"
-        ]
-    },
+  "Nomad": {
```

Comment:
Can you please re-indent this to match the rest of the file? We're using 4 spaces for indentation here